databases/mysql8/Portfile

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    mysql8
+version                 8.0.16
+set boost_version       1.69.0
+categories              databases
+platforms               darwin
+license                 GPL-2
+maintainers             {gmail.com:herby.gillot @herbygillot} openmaintainer
+homepage                http://www.mysql.com/
+
+# Set revision_client and revision_server to 0 on version bump.
+set revision_client 0
+set revision_server 0
+
+set name_mysql          ${name}
+set version_branch      [join [lrange [split ${version} .] 0 1] .]
+set boost_distver       [join [split ${boost_version} .] _]
+set boost_distname      boost_${boost_distver}
+
+
+if {$subport eq $name} {
+    PortGroup           muniversal 1.0
+    PortGroup           cmake 1.1
+    PortGroup           select 1.0
+
+    description         Multithreaded SQL database server
+    long_description    MySQL is an open-source, multi-threaded SQL database.
+    license             GPL-2
+
+    revision            ${revision_client}
+
+    master_sites        mysql:MySQL-${version_branch}:mysql \
+                        sourceforge:project/boost/boost/${boost_version}:boost
+
+    distname            mysql-${version}
+
+    distfiles           ${distname}${extract.suffix}:mysql \
+                        ${boost_distname}${extract.suffix}:boost
+
+    checksums           ${distname}${extract.suffix} \
+                        rmd160  2e0d3326331b74fec8ab3e147f330d2d4f75382e \
+                        sha256  8d9fe89920dc8bbbde2857b7b877ad2fa5ec2f231c68e941d484f3b72735eaea \
+                        size    146811037 \
+                        ${boost_distname}${extract.suffix} \
```

Comment:
I don’t think we want this port to be fetching its own private boost version. Why can it not use the macports provided boost  instead ?

Reply:
(resolved as per discussion)